186 alert value modal

elauria opened the pull request.

hjacobs commented on zmon-controller-ui/views/alertDetails.html:
```diff
@@ -156,7 +156,7 @@
                                 </td>
                                 <td>
-                                    <div class="ellipsis" expand>{{entityInstance.result.value}}
+                                    <div class="ellipsis" alert-value-modal name="entityInstance.entity" value="entityInstance.result.value">{{entityInstance.result.value}}
```
can you check the performance of this?

elauria replied:
it should be ok, since the modal controller and the $watch for 'filter' will only be created after the user click, destroyed on close.

In other words there should only be one modal controller and one filter $watch at a given time (I tested this).

hjacobs commented:
There are two issues for me with the modal window (see screenshot below):

* there is an extra pair of square brackets around the result (I only returned a single JSON object, but the modal result view shows it wrapped in a list)
* we waste some vertical screen space in the modal window (unused white area below the code editor box)

![zmon-result-modal](https://cloud.githubusercontent.com/assets/510328/16260036/b427c632-3865-11e6-9235-9a30b194a33c.png)

elauria commented:
@hjacobs silly of me not to think that the line-height would affect the editor height, but anyways found the right solution.

issue 1) thats the output of jsonPath, always brackets everything except 'false'. Added code to handle this.

issue 2) solved by removing the line limit in the directive, which had nothing to do there and its not needed, and added the proper css classes to the modal and the editor that allow the editor to fit its container.

hjacobs commented:
:+1:

elauria commented:
👍
